common/config/rush/command-line.json

```diff
@@ -65,6 +65,14 @@
       "shellCommand": "rush install --variant stable",
       "safeForSimultaneousRushProcesses": true
     },
+    {
+      "commandKind": "global",
+      "name": "build:all",
```

Comment:
I'd suggest we name this `build:all-flavors`. Currently it is confusing if this means build all packages? (similar to make file `make all`)